.github/tests.yaml

```diff
@@ -0,0 +1,31 @@
+betelgeuse: "tests/foreman/ -v -m 'build_sanity'"
+broker[docker]: "tests/foreman/ -v -m 'build_sanity'"
```

Comment:
Not all tests are exercising the broker here, instead, I would rather run some API tests `factory_instance` or/and `content_host` markers from any specific test module.

Reply:
any example would really help to understand here.

Reply:
I'm running a sanity check here because if we make this too specific to a particular test, there's a risk that the test could become obsolete or even removed altogether. By keeping it more general, we ensure that our approach remains relevant and adaptable to changing circumstances.

Reply:
agree so u can certainly target a test module with that markers associated with it so that its just not one test and even if they remove it we will come to know automatically:

```
pytest -m "factory_instance or content_host" tests/foreman
```

See if u find any tests with `or` operand as well, even if we have one that would be the best test.